### PR TITLE
8289605: Robot capture tests fail intermittently on Mac M1

### PR DESCRIPTION
**Pull request by andy-goryachev-oracle**
- moving mouse pointer to stage lower right corner in order to avoid interference with the Robot screen capture.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8289605](https://bugs.openjdk.org/browse/JDK-8289605): Robot capture tests fail intermittently on Mac M1

### Reviewers
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx pull/854/head:pull/854` \
`$ git checkout pull/854`

Update a local copy of the PR: \
`$ git checkout pull/854` \
`$ git pull https://git.openjdk.org/jfx pull/854/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 854`

View PR using the GUI difftool: \
`$ git pr show -t 854`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/854.diff">https://git.openjdk.org/jfx/pull/854.diff</a>

</details>

### CODE REVIEW

File: tests/system/src/test/java/test/robot/javafx/scene/RobotTest.java
Comment (Maran23) [inline tests/system/src/test/java/test/robot/javafx/scene/RobotTest.java:148]: Since there is only one line to be executed in this lambda, this can also be written as one line.
-> `Util.runAndWait(() -> robot.mouseMove(x,y));`

File: tests/system/src/test/java/test/robot/javafx/scene/layout/RegionBackgroundImageUITest.java
Comment (Maran23) [inline tests/system/src/test/java/test/robot/javafx/scene/layout/RegionBackgroundImageUITest.java:50]: same here.

File: tests/system/src/test/java/test/robot/javafx/scene/layout/RegionBackgroundImageUITest.java
Comment (Maran23) [inline tests/system/src/test/java/test/robot/javafx/scene/layout/RegionBackgroundImageUITest.java:25]: Looks like you forgot to modify the copyright year above. 😄

File: tests/system/src/test/java/test/robot/javafx/scene/layout/RegionBackgroundImageUITest.java
Comment (kevinrushforth) [inline tests/system/src/test/java/test/robot/javafx/scene/layout/RegionBackgroundImageUITest.java:47]: You might consider moving this method to the `VisualBase` superclass, since it would be possible for any other robot-based screen capture test to fail in the future. If you do that, you wouldn't have a stage that you could access, so you would need to do something like move the mouse to the lower right of the screen rather than the window.

**Review by kevinrushforth — COMMENTED** [review]
I expect that the fix will work, so I'll test it. I left a comment inline with a suggestion that should make it less likely for other tests to have the same problem in the future.

**Comment by kevinrushforth** [comment]
The fix works on my M1 system for the two failing tests. So my only feedback is the suggestion to move the `before` method from `RegionBackgroundImageUITest` to `VisualTestBase`.

**Comment by kevinrushforth** [comment]
> VisualTestBase is a base class to the whole gallery of tests, some of which have more than one stage (IconifyTest). I've limited the changes to RegionBackgroundImageUITest because I did not want to expand the scope unnecessarily.

OK.

**Review by kevinrushforth — APPROVED** [review]
Looks good.

You could simplify the lambda as @Maran23 suggests, but I'll leave that up to you. I'll re-approve if you do.

**Pull request closed without merging**
